clients/web/static/lib/bootstrap/css

```diff
@@ -1 +0,0 @@
-../../built/
```

Comment:
Why are we removing this? @kotfic just put in a fix to preserve compatibility with the old path.